OpenRA.Mods.RA/Traits/SupportPowers/GpsPower.cs

```diff
@@ -31,17 +31,22 @@ class GpsPowerInfo : SupportPowerInfo
 		[SequenceReference("SatelliteImage")] public readonly string SatelliteSequence = "idle";
 		[PaletteReference] public readonly string SatellitePalette = "effect";
 
+		[Desc("Requires an actor with `ProvidesRadar` to show GPS dots.")]
+		public readonly bool RequiresRadar = true;
```

Comment:
You are misunderstanding what this is doing. I should probably call it `RequiresActiveRadar`.